app/components/task_list/check_box_action.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="govuk-checkboxes__item">
+  <%= hidden_field_tag(@name, "0", {id: nil}) %>
+  <%= check_box_tag(@name, 1, @task.send(@attribute), {class: "govuk-checkboxes__input"}) %>
```

Comment:
It's a shame that we can no longer use the Gov form builder!

Reply:
We can still use the builder on the forms, the intention here is that the component makes composing a common task view simple. 

I never liked the component calling the builder. I guess the longer solution is to extned the builder for our version of a check box action, but that felt like a bigger piece of work.